openedx_external_enrollments/tests/external_enrollments/tests_salesforce_external_enrollment.py

```diff
@@ -219,6 +222,62 @@ def test_get_program_of_interest_data(self, get_course_mock, get_user_mock):
         self.assertEqual(expected_data, program_data)
         self.assertTrue(drupal_id.startswith(expected_drupal))
 
+    @patch(
+        'openedx_external_enrollments.external_enrollments.salesforce_external_enrollment.ProgramSalesforceEnrollment'
+    )
+    @patch('openedx_external_enrollments.external_enrollments.salesforce_external_enrollment.get_user')
+    @patch.object(SalesforceEnrollment, '_get_course')
+    def test_get_program_of_interest_data_from_program(self, get_course_mock, get_user_mock, model_mock):
```

Comment:
@Jacatove, I see there are two unit tests cases here, could you split it up, please?